app/models/request.rb

```diff
@@ -110,8 +110,7 @@ def proxy?
   def notification_email_address
     if proxy? && user.proxy_access.email_address.present?
       user.proxy_access.email_address
-    elsif user.email_address.present?
-      user.email_address
+    else user.email_address
     end
```

Comment:
This reads a little weird (and TBH, I'm not sure what the behavior is for this).

Would you mind putting `user.email_address` on a separate line?